apiary.apib

```diff
@@ -542,7 +542,186 @@ API endpoints that Operation Code's Rails backend makes available to its React f
+                "requested_mentor_id": 250,
+                "status": "Open",
+                "assigned_mentor_id": 250
+            }
+
++ Response 201 (application/json)
+
+    + Body
+
+            {   
+                "id": 2130,
+                "user_id": 4600,
+                "details": "I want to type well",
+                "language": "English",
+                "service_id": 200,
+                "requested_mentor_id": 250,
+                "status": "Open",
+                "assigned_mentor_id": 250
+            }
+
++ Response 422 (application/json)
+
+    + Body
+
+            {
+                errors: "Some error message"
+            }
+  
+## Mentor Request | Show [/api/v1/requests/{request_id}]
++ Parameters
+    
+    + request_id (integer, required) - Indentification of mentor request in the form of an integer
+
+    
+### Show an Exiting Request From User ID [POST]
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+
+    + Body
+
+            {
+                "request_id": 1234
+
+            }
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                "request_id": 1234,
+                "user_id": 5200,
+                "details": "I want to program",
+                "language": "English",
+                "service_id": 210,
+                "requested_mentor_id": 260,
+                "status": "Closed",
+                "assigned_mentor_id": 260
+            }
+
+## Mentor Requests | Unclaimed Collection [/api/v1/requests]
+
+### Get All Unclaimed Mentor Requests [GET]
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+        
++ Response 200 (application/json)
+
+        [
+            {   
+                "id": 251,
+                "service_id": 223,
+                "language": "English",
+                "details": "I want to stop eating crayons",
+                "user_id": 1113,
+                "assigned_mentor_id": null,
+                "requested_mentor_id": 3333,
+                "created_at": "2017-09-26T06:54:58.607Z",
+                "updated_at": "2017-09-26T06:54:58.607Z",
+                "status": "open"            
+            },
+            {   "id":"264"
+                "service_id": 243,
+                "language": "American Sign Language",
+                "details": "I need help interviewing",
+                "user_id": 2500,
+                "assigned_mentor_id": null,
+                "requested_mentor_id": 1250,
+                "created_at": "2017-09-26T06:54:58.607Z",
+                "updated_at": "2017-09-26T06:54:58.607Z",
+                "status": "open"            
+            }
+        ]
+        
++ Response 401 (application/json)
+
+        {
+            error: 'Unauthorized'
+        }
+
+## Mentor Request | Update [/api/v1/requests{user_id}{?details,language,service_id,requested_mentor_id,status,assigned_mentor_id}]
```

Comment:
There will be a `/` after requests because the `id` is part of the url itself, i.e. `/api/v1/requests/{user_id}{?details,language....`